index.js

```diff
@@ -688,25 +688,38 @@ prototype.buildModel = function(oldModel) {
   var properties = {};
 
   this.forEach(oldModel.properties, function(oldProperty, propertyName) {
+    var allowedProperties = {
+      description: oldProperty.description
+    };
+
     if (fixNonStringValue(oldProperty.required) === true) {
       required.push(propertyName);
     }
 
+    if (oldProperty.hasOwnProperty('example')) {
+      allowedProperties.example = oldProperty.example;
+    }
+
     properties[propertyName] = extend({},
       this.buildDataType(oldProperty, true),
-      {description: oldProperty.description}
+      allowedProperties
```

Comment:
That will put an undefined value for the example key, no?

On Thursday, November 19, 2015, Ivan Goncharov <notifications@github.com>
wrote:

> In index.js
> <https://github.com/apigee-127/swagger-converter/pull/62#discussion_r45416240>
> :
>
> >      properties[propertyName] = extend({},
> >        this.buildDataType(oldProperty, true),
> > -      {description: oldProperty.description}
> > +      allowedProperties
>
> You can see check here
> https://github.com/apigee-127/swagger-converter/blob/master/index.js#L824
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apigee-127/swagger-converter/pull/62/files#r45416240>.
>


-- 


  Steve Bussey •SalesLoft

•Software Engineer

e steve.bussey@salesloft.com w salesloft.com
<http://salesloft.com/?v=1&utm_expid=76406993-1.E5ncJhwyTAW9uGaR1WdBTA.1?utm_source=Email-Signature-Rescue&utm_medium=Email-Signature&utm_campaign=Email-Signature-Rescue>

a 3423 Piedmont Rd NE, Atlanta, GA 30305

[image: Facebook] <https://www.facebook.com/SalesLoft> [image: Twitter]
<https://twitter.com/SalesLoft> [image: Linkedin]
<https://www.linkedin.com/company/salesloft> [image: wordpress]
<http://blog.salesloft.com/blog/> [image:
http://salesloft.com/prospector/#video]
<http://salesloft.com/prospector/#video>  [image: Software Engineer Logo]
<http://salesloft.com/?v=1&utm_expid=76406993-1.E5ncJhwyTAW9uGaR1WdBTA.1?utm_source=Email-Signature-Rescue&utm_medium=Email-Signature&utm_campaign=Email-Signature-Rescue>